### ISSUE TITLE
Unclear User Understanding of App

### ISSUE THREAD

**Issue by slenkala** [KilimAnnejaro/CS506FinalProject#8]
**Summary**: Unclear user understanding of App

**Description**: When the app opens, the login screen is the only unambiguous part. Once you are on the home screen, the purpose of the app is unclear. Because we have read your design review and have background regarding your app, we were able to navigate it. However, for a new user, they can easily be confused. For example, a list of recommended candidates shows without you entering any information. Why is this list here? Am I supposed to do something with this list? In addition, when you navigate to the ranking tab, it seems like the buttons are static and no matter what you input, the recommendations don’t change (https://github.com/KilimAnnejaro/CS506FinalProject/issues/9). Also, it is unclear what the “Top Issues” component is. Are these your top issues? Are theses issues are you able to interact with? Are these issues related to a specific candidate? Right now it just seems like a static arbitrary list of issues. 

**Reproduction Steps**: Login to the app > See the recommended candidates and top issues > Now what? > Go to rankings tab by guess, then understand you are suppose to rank candidates > Click opinions on different candidates > See no effect in recommended candidates > Confused.

**Severity**: High

**Suggestions(optional)**: We suggest changing how your app flows for a first time user to clear up ambiguities. For example, once you log-in, the first page a user see’s should be an instruction page or a page where user can input their opinions. Then, only after a user inputs their opinions, should they be able to see their recommended candidates. We are unsure what the top issues component is suppose to achieve, so once that ambiguity is cleared up, we can give a recommendation on how it fits in.

**Comment by parlough** [comment]
In your documents you also described an info/help page which would also help contribute to solving this issue. You mentioned it is lower priority, but due to your setup with React, I can imagine this would be a fairly simple addition as you described it as mainly having text information. So I feel it would be worth adding quickly before your final demo. Some pictures could help as well within there, but aren't necessary.
